en/deploy-on-aws-eks.md

```diff
@@ -300,485 +133,301 @@ You can use the `terraform output` command to get the output again.
-  pd_instance_type              = "c5.large"
-  tikv_count                    = 1
-  tikv_instance_type            = "c5d.large"
-  tidb_count                    = 1
-  tidb_instance_type            = "c4.large"
-  monitor_instance_type         = "c5.large"
-  create_tidb_cluster_release   = false
-}
-```
+Some AWS instance types provides additional [NVMe SSD local store volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html). You can choose such instances for the TiKV node pool to achieve higher IOPS and lower latency.
 
 > **Note:**
 >
-> The `cluster_name` of each cluster must be unique.
-
-When you finish the modification, execute `terraform init` and `terraform apply` to create the nodes pool for the TiDB cluster.
+> During the EKS upgrade, data in the local storage will be [lost](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-lifetime) due to the node reconstruction. when the node reconstruction happens, you need to migrate data in TiKV. If you do not want to migrate data, it is recommended not to use the local disk in the production environment.
 
-Finally, [deploy TiDB cluster and monitor](#deploy-tidb-cluster-and-monitor).
+For instance types that provide local volumes, see [AWS Instance Types](https://aws.amazon.com/ec2/instance-types/). Take `c5d.4xlarge` as an example:
 
-## Destroy clusters
+1. Create a node group with local storage for TiKV.
 
-1. [Destroy the TiDB clusters](destroy-a-tidb-cluster.md).
+    Modify the instance type of the TiKV node group in the `eksctl` configuration file to `c5d.4xlarge`:
 
-2. Destroy the EKS cluster by the following command:
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    terraform destroy
     ```
-
-    If the following error occurs during `terraform destroy`:
-
-    ```
-    Error: Get http://localhost/apis/apps/v1/namespaces/kube-system/deployments/tiller-deploy: dial tcp [::1]:80: connect: connection refused
+      - name: tikv
+        instanceType: c5d.4xlarge
+        labels:
+          role: tikv
+        taints:
+          dedicated: tikv:NoSchedule
+        ...
     ```
 
-    Run the following command:
+    Create the node group:
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    terraform state rm module.tidb-operator.helm_release.tidb-operator
+    eksctl create nodegroups -f cluster.yaml
     ```
 
-    And then run `terraform destroy` again.
-
-> **Note:**
->
-> * This will destroy your EKS cluster.
-> * If you do not need the data on the volumes anymore, you have to manually delete the EBS volumes in AWS console after running `terraform destroy`.
-
-## Manage multiple Kubernetes clusters
-
-This section describes the best practice to manage multiple Kubernetes clusters, each with one or more TiDB clusters installed.
+    If the tikv node group already exists, you can either delete the old group and then create a new one, or change the group name to avoid conflict.
```

Comment:
```suggestion
    If the TiKV node group already exists, you can either delete the old group and then create a new one, or change the group name to avoid conflict.
```